pygmo/_py_algorithms.py

```diff
@@ -0,0 +1,131 @@
+import random
+
+try:
+    from scipy.optimize import minimize
+
+    class scipy:
+        """
+        This class is a user defined algorithm (UDA) providing a wrapper around the function scipy.optimize.minimize.
+        The constructor accepts those arguments that are specific to the algorithm:
+        - args
+        - method
+        - tol - the tolerance
+        - callback
+        - options
+
+        Other aspects, like bounds or the existence of a gradient and hessian, are taken from the problem.
```

Comment:
Other aspects, like bounds or the existence of a gradient and hessian, are taken from the problem.
->
The problem bounds and the existence of a gradient and hessian, are deduced calling the relevant problem methods (problem.has_gradient(), etc..)

Reply:
Updated the phrasing.